core/optaplanner-core-impl/src/main/java/org/optaplanner/core/config/heuristic/selector/move/generic/list/ListKOptMoveSelectorConfig.java

```diff
@@ -0,0 +1,61 @@
+package org.optaplanner.core.config.heuristic.selector.move.generic.list;
+
+import java.util.function.Consumer;
+
+import javax.xml.bind.annotation.XmlType;
+
+import org.optaplanner.core.config.heuristic.selector.move.MoveSelectorConfig;
+import org.optaplanner.core.config.util.ConfigUtils;
+
+@XmlType(propOrder = {
+        "minimumK",
+        "maximumK",
+})
+public class ListKOptMoveSelectorConfig extends MoveSelectorConfig<ListKOptMoveSelectorConfig> {
+
+    public static final String XML_ELEMENT_NAME = "listKOptMoveSelector";
+
+    protected Integer minimumK = null;
+    protected Integer maximumK = null;
+
+    public Integer getMinimumK() {
+        return minimumK;
+    }
+
+    public void setMinimumK(Integer minimumK) {
+        this.minimumK = minimumK;
+    }
+
+    public Integer getMaximumK() {
+        return maximumK;
+    }
+
+    public void setMaximumK(Integer maximumK) {
+        this.maximumK = maximumK;
+    }
```

Comment:
Do we want to add the appropriate `with...()` methods?
Some configs have them, some not, I'm wondering if we perhaps want to make it the new standard.